site/src/lib/api/blocks/npm.ts

```diff
@@ -90,19 +82,6 @@ const mirrorNpmPackageToR2 = async (
     });
   }
 
-  let metadataJson;
```

Comment:
A bunch of metadata checking stuff has been moved inside `r2` where it can be shared across all publishing methods, but otherwise unchanged